tests/test_task_download.py

```diff
@@ -35,6 +35,24 @@ def test_download_item_asset(tmp_path: Path, item_collection: Dict[str, Any]) ->
     assert filename.is_file() is True
 
 
+# @vcr.use_cassette(str(cassettepath / 'download_assets'))
```

Comment:
What's the deal with the commented `@vcr`?

Reply:
Oh, that, that's my love of delicious copy-pasta.  Will punt, make some other clean up, and re-request.

Reply:
Given you already approved, I now realize re-requesting would be silly.